openquake/hazardlib/tests/gsim/germany_2018_test.py

```diff
@@ -0,0 +1,143 @@
+        self.imts = [PGA(), SA(0.1), SA(0.2), SA(0.5), SA(1.0), SA(2.0)]
+        self.mags = [4.5, 5.5, 6.5, 7.5]
+        self.rakes = [-90., 0., 90.]
+        self.dctx = DistancesContext()
+        self.dctx.rhypo = np.array([5., 10., 20., 50., 100.])
+        self.sctx = SitesContext()
+        self.sctx.vs30 = 800.0 * np.ones(5)
+
+    def check_gmpe_adjustments(self, adj_gmpe_set, original_gmpe):
+        """
+        Takes a set of three adjusted GMPEs representing the "low", "middle"
+        and "high" stress drop adjustments for Germany and compares them
+        against the original "target" GMPE for a variety of magnitudes
+        and styles of fauling.
+        """
+        low_gsim, mid_gsim, high_gsim = adj_gmpe_set
+        tot_std = [const.StdDev.TOTAL]
+        for imt in self.imts:
+            for mag in self.mags:
+                for rake in self.rakes:
+                    rctx = RuptureContext()
+                    rctx.mag = mag
+                    rctx.rake = rake
+                    rctx.hypo_depth = 10.
+                    # Get "original" values
+                    mean = original_gmpe.get_mean_and_stddevs(self.sctx, rctx,
+                                                              self.dctx, imt,
+                                                              tot_std)[0]
+                    mean = np.exp(mean)
+                    # Get "low" adjustments (0.75 times the original)
+                    low_mean = low_gsim.get_mean_and_stddevs(self.sctx, rctx,
+                                                             self.dctx, imt,
+                                                             tot_std)[0]
+                    np.testing.assert_array_almost_equal(
+                        np.exp(low_mean) / mean, 0.75 * np.ones_like(low_mean))
+
+                    # Get "middle" adjustments (1.25 times the original)
+                    mid_mean = mid_gsim.get_mean_and_stddevs(self.sctx, rctx,
+                                                             self.dctx, imt,
+                                                             tot_std)[0]
+                    np.testing.assert_array_almost_equal(
+                        np.exp(mid_mean) / mean, 1.25 * np.ones_like(mid_mean))
+
+                    # Get "high" adjustments (1.5 times the original)
+                    high_mean = high_gsim.get_mean_and_stddevs(self.sctx, rctx,
+                                                               self.dctx, imt,
+                                                               tot_std)[0]
+                    np.testing.assert_array_almost_equal(
+                        np.exp(high_mean) / mean,
+                        1.5 * np.ones_like(high_mean))
+
+    def test_akkar_germany_adjustments(self):
+        adj_gmpes = [AkkarEtAlRhyp2014LSD(),
+                     AkkarEtAlRhyp2014MSD(),
+                     AkkarEtAlRhyp2014HSD()]
+        self.check_gmpe_adjustments(adj_gmpes, AkkarEtAlRhyp2014())
```

Comment:
Having a single class `AkkarEtAlRhyp2014` the change would be
```python
 adj_gmpes = [AkkarEtAlRhyp2014("L"),
￼                      AkkarEtAlRhyp2014("M"),
￼                      AkkarEtAlRhyp2014("H")]
```
and similarly for the tests below for the other classes.